docs/source/adb.rst

```diff
@@ -0,0 +1,136 @@
+.. _adb:
```

Comment:
I am not seeing this page in the left pane or on the index page. I think it should. Was this intentional?